pkg/reconciler/service/service.go

```diff
@@ -186,8 +187,9 @@ func (c *Reconciler) checkRoutesNotReady(config *v1.Configuration, logger *zap.S
 	}
 }
 
-func (c *Reconciler) createConfiguration(service *v1.Service) (*v1.Configuration, error) {
-	cfg, err := resources.MakeConfiguration(service)
+func (c *Reconciler) createConfiguration(ctx context.Context, service *v1.Service) (*v1.Configuration, error) {
+	gc := cfgmap.FromContextOrDefaults(ctx).Features.ResponsiveRevisionGC
+	cfg, err := resources.MakeConfigurationFromExisting(service, &v1.Configuration{}, gc)
```

Comment:
Leave this as  `MakeConfiguration` ?

Reply:
We are using this override so we can still pass the GC flag. Once we clean up that flag we can go back to the regular MakeConfiguration.

Reply:
thanks for clarifying that v1 needs to add the RouteLabel